cmd/generic-handlers.go

```diff
@@ -592,7 +593,7 @@ func (h pathValidityHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 func setRateLimitHandler(h http.Handler) http.Handler {
 	_, maxLimit, err := sys.GetMaxOpenFileLimit()
 	if err != nil {
-		panic(err)
+		logger.FatalIf(err, "")
```

Comment:
err != nil check is redundant and should be removed.

Reply:
Done. Removed the unnecessary check.